weather_dl/download_pipeline/pipeline_test.py

```diff
@@ -56,7 +56,8 @@
                                   num_requests_per_key=-1,
                                   partition_chunks=None,
                                   schedule='in-order',
-                                  check_skip_in_dry_run=False),
+                                  check_skip_in_dry_run=False,
+                                  update_manifest=False),
```

Comment:
Let's add a(n integration) test either here or in the partitions_tests file that exercises the update manifest feature.

Reply:
Added unit tests in partitions_tests.